Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -58,7 +58,7 @@ ref class AppEventListener
 
 extern "C" void _ApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg);
```

Comment:
This is not an exported function but is defined in the header here for the source file to use without having to extern it

---
In reply to: [72347990](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72347990) [](ancestors = 72347990,72336476)